elyra/pipeline/component.py

```diff
@@ -271,7 +271,7 @@ def read_component_definition(self, component_id: str, location: str) -> str:
         res = requests.get(location)
```

Comment:
Same as previous - please update the return type hint to `-> Optional[str]:` since `None` can now be returned.